.github/workflows/presubmit.yml

```diff
@@ -0,0 +1,54 @@
+name: presubmit
+
+on: pull_request
+
+jobs:
+  lint-js:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install ESLint with Google Config
+      run: npm install eslint@^7.3.1 eslint-config-google@^0.14.0
+    - name: Check
+      run: npx eslint src
```

Comment:
Seems we don't have "src" directory, so the linter can't find the files, consider adjust the directory structure.

Reply:
sorry for the late response! . on the repo main page I see the "src" , so i confused on how to solve this problem

Reply:
This isn't an issue with the directory structure as the error would seem to indicate, it's returning an error as there are no javascript files in this directory. Adding an empty javascript file will resolve this issue. It might be better to change the path to src/main/webapp as this will reduce the search space for the command.

Reply:
For reference one can use this tool https://github.com/nektos/act along with installing docker to test these github actions locally. That is how I was able to discover this was the issue/